### PR TITLE
Created June updates and modified May update a bit

### PR DESCRIPTION
**Pull request by erogluorhan**

### CODE REVIEW

File: blog/_posts/2021-07-01-June-2021-update.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Please see 
+[GeoCAT-f2py v2021.06.0 Release](https://github.com/NCAR/geocat-f2py/releases/tag/v2021.06.0)
+for further information about Release and Developer Notes.
+
+- Please see our 
+[Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) 
+for detailed information about our development stack. 
+
+## Community Events
+
```
Comment (michaelavs) [inline blog/_posts/2021-07-01-June-2021-update.md:39]:
```suggestion

```

File: blog/_posts/2021-07-01-June-2021-update.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Please see 
+[GeoCAT-f2py v2021.06.0 Release](https://github.com/NCAR/geocat-f2py/releases/tag/v2021.06.0)
+for further information about Release and Developer Notes.
+
+- Please see our 
+[Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) 
+for detailed information about our development stack. 
+
+## Community Events
```
Comment (michaelavs) [inline blog/_posts/2021-07-01-June-2021-update.md]:
```suggestion
### - Community Events

```

**Review by michaelavs — APPROVED** [review]
Looks good to me! Just a couple of minor format things and then I think it can be merged!

File: blog/_posts/2021-07-01-June-2021-update.md (hunk 2 above)
Comment (erogluorhan) [reply blog/_posts/2021-07-01-June-2021-update.md]: Thanks for this suggestion; it led me rethink about the sizes and fonts of all the titles, and I have applied a revision to all of them for May and June updates. If we like them, I will do the same change for previous monthly updates as well.

**Pull request merged**